.github/workflows/api-checks.yml

```
@@ -0,0 +1,55 @@
+name: Template API Checks
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - app/**
+      - .github/workflows/api-checks.yml
+  pull_request:
+    paths:
+      - app/**
+      - .github/workflows/api-checks.yml
+
+defaults:
+  run:
+    working-directory: ./app
+
+env:
+  # Some commands in the Makefile check if this is set
+  # and add additional parameters to the commands called
+  CI: true
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: build container
```

nit: use sentence case to be consistent with other step names
```suggestion
      - name: Build container
```